plypatch/__init__.py

```diff
@@ -295,7 +295,8 @@ def _ensure_name_and_email_set(self):
             raise exc.GitConfigRequired('user.name')
 
     def restore(self, three_way_merge=True, commit_msg=None,
-                fetch_remotes=True, customize_commit_msg=False):
+                fetch_remotes=True, customize_commit_msg=False,
+                no_commit=False):
```

Comment:
no_commit makes a bit of a double-negative when using logical operators. How about skip_commit?

Reply:
I actually had skip_commit first, but renamed it to no_commit because I like `--no-commit` as the CLI arg better (there's already some "skip" terminology in ply, and I'd rather avoid confusion). Also, it seems the same amount of negativity... 😐